### PR TITLE
[fix]: [PL-25096]: Adding description of all exposed APIs in Framework Squad 

### PR DESCRIPTION
**Pull request by shreyas-harness**
## Describe your changes
Improved description to framework squad APIs
## Checklist
- [x] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/34244)

### CODE REVIEW

**Comment by shreyas-harness** [comment]
trigger messagecheck

**Comment by shreyas-harness** [comment]
trigger messagecheck

**Comment by shreyas-harness** [comment]
trigger messagecheck

File: 120-ng-manager/src/main/java/io/harness/ng/core/remote/OrganizationResource.java [hunk 1]
```diff
@@ -158,8 +160,8 @@ public class OrganizationResource {
 
   @GET
   @ApiOperation(value = "Get Organization list", nickname = "getOrganizationList")
-  @Operation(operationId = "getOrganizationList",
-      summary = "Get the list of Organizations satisfying the criteria (if any) in the request",
+  @Operation(operationId = "getOrganizationList", summary = "Get list of Organizations",
```
Comment (ashishsanodia-harness) [inline 120-ng-manager/src/main/java/io/harness/ng/core/remote/OrganizationResource.java:163]: should be "List Organizations"

File: 120-ng-manager/src/main/java/io/harness/ng/core/remote/OrganizationResource.java [hunk 2]
```diff
@@ -136,7 +137,8 @@ public class OrganizationResource {
   @GET
   @Path("{identifier}")
   @ApiOperation(value = "Get an Organization by identifier", nickname = "getOrganization")
-  @Operation(operationId = "getOrganization", summary = "Get the Organization by accountIdentifier and orgIdentifier",
+  @Operation(operationId = "getOrganization", summary = "Get organization",
```
Comment (ashishsanodia-harness) [inline 120-ng-manager/src/main/java/io/harness/ng/core/remote/OrganizationResource.java:140]: O should be CAPS

File: 120-ng-manager/src/main/java/io/harness/ng/core/remote/ProjectResource.java [hunk 3]
```diff
@@ -173,7 +174,7 @@ public class ProjectResource {
 
   @GET
   @ApiOperation(value = "Get Project list", nickname = "getProjectList")
-  @Operation(operationId = "getProjectList", summary = "List user's project",
+  @Operation(operationId = "getProjectList", summary = "List project", description = "List user's project",
```
Comment (ashishsanodia-harness) [inline 120-ng-manager/src/main/java/io/harness/ng/core/remote/ProjectResource.java:177]: should be "List Projects"

File: 120-ng-manager/src/main/java/io/harness/ng/core/remote/ProjectResource.java [hunk 4]
```diff
@@ -289,7 +290,8 @@ public class ProjectResource {
   @DELETE
   @Path("{identifier}")
   @ApiOperation(value = "Delete a Project by identifier", nickname = "deleteProject")
-  @Operation(operationId = "deleteProject", summary = "Deletes the Project corresponding to the specified Project ID.",
+  @Operation(operationId = "deleteProject", summary = "Delete project",
```
Comment (ashishsanodia-harness) [inline 120-ng-manager/src/main/java/io/harness/ng/core/remote/ProjectResource.java:293]: Change project to Project

File: 400-rest/src/main/java/software/wings/resources/secretsmanagement/AzureSecretsManagerResource.java [hunk 5]
```diff
@@ -45,7 +46,10 @@ public class AzureSecretsManagerResource {
   @Inject AzureSecretsManagerServiceImpl azureSecretsManagerService;
 
   @POST
-  public RestResponse<String> saveAzureSecretsManagerConfig(
+  @Operation(
+      summary = "Save azure secrets manager config", description = "Saves the Azure secrets manager configuration")
```
Comment (ashishsanodia-harness) [inline 400-rest/src/main/java/software/wings/resources/secretsmanagement/AzureSecretsManagerResource.java:50]: should be Azure

File: 400-rest/src/main/java/software/wings/resources/secretsmanagement/AzureSecretsManagerResource.java [hunk 6]
```diff
@@ -64,6 +71,7 @@ public RestResponse<Boolean> deleteAzureVaultConfig(
 
   @POST
   @Path("list-vaults")
+  @Operation(summary = "List azure vault", description = "Lists azure vaults found for the account id")
```
Comment (ashishsanodia-harness) [inline 400-rest/src/main/java/software/wings/resources/secretsmanagement/AzureSecretsManagerResource.java:74]: should be Azure

File: 400-rest/src/main/java/software/wings/resources/secretsmanagement/SecretManagerResourceNG.java [hunk 7]
```diff
@@ -76,8 +85,10 @@ public RestResponse<ConnectorValidationResult> validateSecretManager(@PathParam(
 
   @GET
   @Path("{identifier}")
-  public RestResponse<SecretManagerConfigDTO> getSecretManager(
-      @PathParam(NGCommonEntityConstants.IDENTIFIER_KEY) String identifier,
+  @Operation(operationId = "getSecretManager", summary = "Get secret manager",
```
Comment (ashishsanodia-harness) [inline 400-rest/src/main/java/software/wings/resources/secretsmanagement/SecretManagerResourceNG.java:88]: Be consistent with CASES like above "Validate Secret maanger" but here it's "Get secret manager"

File: 440-connector-nextgen/src/main/java/io/harness/connector/apis/resource/ConnectorResource.java [hunk 8]
```diff
@@ -443,7 +448,8 @@ public ConnectorResource(@Named("connectorDecoratorService") ConnectorService co
   @GET
   @Path("catalogue")
   @ApiOperation(value = "Get Connector Catalogue", nickname = "getConnectorCatalogue")
-  @Operation(operationId = "getConnectorCatalogue", summary = "Gets the Connector catalogue by Account Identifier",
+  @Operation(operationId = "getConnectorCatalogue", summary = "Gets Connector catalogue",
```
Comment (ashishsanodia-harness) [inline 440-connector-nextgen/src/main/java/io/harness/connector/apis/resource/ConnectorResource.java:451]: be consistent with Get over Gets

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 6 Issues 
* Vulnerability 2 Vulnerabilities 
 * Code issue 4 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/7752dbb3099d2dba3b5d37f5931a5e4c/pull-request/34244)

**Pull request closed without merging**